conformance/lang/expressions/relational-expr/greater_than_expression.balt

```diff
@@ -0,0 +1,1253 @@
+   io:println(c); // @output false
+}
+
+Test-Case: output
+Description: Test type cast expression as greater than expression operands.
+Labels: DecimalNumber, int, relational-expr, type-cast-expr
+
+function init() {
+   boolean i = <int> 6 > <int> 7.5;
+   io:println(i); // @output false
+}
+
+Test-Case: output
+Description: Test shift expressions as greater than expression operands.
+Labels: DecimalNumber, int, relational-expr, shift-expr
+
+function init() {
+   boolean p = (1 << 2) > 6;
+   io:println(p); // @output false
+}
+
+Test-Case: output
+Description: Test binary bitwise expressions as greater than expression operands.
+Labels: binary-bitwise-expr, DecimalNumber, int, relational-expr
+
+function init() {
+   boolean a = (2 & 6) > (2 ^ 6);
+   io:println(a); // @output false
+}
+
+Test-Case: output
+Description: Test if block with greater than expression condition.
+Labels: binary-bitwise-expr, DecimalNumber, int, relational-expr
+
+function init() {
+   if (6 > 5) {
+       io:println("True"); // @output True
+   }
+}
+
+Test-Case: output
+Description: Test binary conditional expressions with greater than expression condition.
+Labels: binary-conditional-expr, DecimalNumber, int, relational-expr
+
+function init() {
+   int a = 12;
+   int b = 15;
+   int c = a > b ? 2 : a;
+   io:println(c); // @output 12
+}
+
+Test-Case: output
+Description: Test greater than expression with list as operands.
+Labels: array-type, Decimal, DecimalFloatingPointNumber, DecimalNumber, int, float, list-constructor-expr, nil-literal, relational-expr, string
+
+function init() {
+   int[] a = [12, 100, 2000000];
+   int[] b = [13, 3];
+   int[] c = [];
+   int[] d = [];
+   string?[] e = ["Hello", "World!", (), "to", "Ballerina"];
+   string?[] f = ["Hello", "World!", (), "from", "Ballerina"];
+   float?[] g = [12.5, 43, ()];
+   float?[] h = [12.5, 43, 4];
+   decimal?[] i = [45, 23.54232, 44];
+   decimal?[] j = [45, 23.54232, 44, ()];
+
+   io:println(a > b); // @output false
+   io:println(c > d); // @output false
+   io:println(c > b); // @output false
+   io:println(h > g); // @output false
+   io:println(i > j); // @output false
+}
+
+Test-Case: output
+Description: Test greater than expression with tuple as operands.
+Labels: Decimal, DecimalFloatingPointNumber, DecimalNumber, int, float, list-constructor-expr, nil-literal, relational-expr, string, tuple-type
+
+function init() {
+   [int, string, float] a = [1, "AB", 33.33];
+   [int, string, float] b = [1, "AB", 34];
+
+   io:println(b > a); // @output true
+
+   [string, string, int?, string] c = ["Hello", "World!", (), "to"];
+   [string, string, int?, string] d = ["Hello", "World!", (), "from"];
+
+   io:println(c > d); // @output true
+
+   [int, float] e = [59215, (0.0 / 0.0)];
+   [int, float] f = [59215, 123.432];
+
+   io:println(e > f); // @output false
+
+   [int, decimal?] g = [59215, 4354.323];
+   [int, decimal?] h = [59215, ()];
+
+   io:println(g > h); // @output false
+}
+
```

Comment:
Let's add some tests with `var` also. 
Ex:
```ballerina
public function main() {
    var a = 10;
    var b = 12;
    io:println(a < b);
}
```
Cover different types as well.
Add the same with other operations too.